schema/nscloud/attachments.proto

```diff
@@ -0,0 +1,13 @@
+// Copyright 2022 Namespace Labs Inc; All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+
+syntax = "proto3";
+
+package foundation.schema.nscloud;
+
+option go_package = "namespacelabs.dev/foundation/schema/nscloud";
+
+message CreatorAttachment {
+    string username = 1;
```

Comment:
I don't think we ever defined what an ns username means. Should we have the user ID here instead?